CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 ### Added 
 
 - Add `NetworkError` enum - [#4](https://github.com/ios-course/simple-network-service/pull/4)
+- Add `HTTPMethod` enum, `HTTPHeaders`, `HTTPTask` enum, `HTTPAPIEndpoint` protocol, `BodyParameters`, `URLParameters` type aliases - [#2](https://github.com/ios-course/simple-network-service/pull/2)
```

Comment:
Please correct this line. 
It looks like the `HTTPHeaders` is an enum.

Reply:
Fixed.